src/test/scala/com/julienvey/trello/CardIt.scala

```diff
@@ -116,4 +119,34 @@ class CardIt extends FunSpec with Matchers {
       val thrown = the [NotFoundException] thrownBy trello.deleteAttachment(cardId, "5c56febb4e84e50254c2c54d")
     }
   }
+
+  describe("Comment API") {
+    it("Add comment to card") {
+      val cardId = "5c4d89b5bd5a2640f5fcb32c"
```

Comment:
what do you think about making this and related tests self-contained, without relying on pre-created data?
it could create a card at the beginning, and also retrieve actionId when needed and use it in the test. no need in keeping cardId and actionId in the repo then.
I am worried if we ever lose the test project, it would be a lot of work to get these integration tests working again.

Reply:
Basically it's a very good theoretical idea. Having a independent test cases is always safe and reliable.
If we should provide the `listId` in which card should be created. So it turns out we using predefined `listId`, should we create list first and then create the list? Dependency chain goes to the project it self.

IMO self-contained test for this particular case does add more complexity rather then benefits. In the other hand if we decide to refactor whole integration test suite that will be great. If you insist to make this test self-contained, I'll do that.

Reply:
pre-creating all data would not be hard - each integration test can check if the required project+list already exist, and if not - launch "initial test setup" code.
that would be a separate improvement though.

for now we can assume that the project and maybe a specific list already exist, and only create cards in tests. this way it is easy to restore test data if it gets lots - just need to manually create one project and one list, rather than multiple cards used in various tests.

Reply:
Looks reasonable to me. I'll provide a fixup for this.